src/components/banner/banner.jsx

```diff
@@ -1,137 +1,151 @@
-					>
-						<div class={style.title}>
-							<LocalLabel localizeKey='title'>Ads help us run this site</LocalLabel>
-						</div>
-						<LocalLabel localizeKey='description'>
-						When you visit our site, pre-selected companies may access and use certain information on your device to serve relevant ads or personalized content.
-						</LocalLabel>
-						<div class={style.info}>
-							<a onClick={this.handleInfo(PANEL_COLLECTED)}>
-								<ChevronIcon class={[style.expand, selectedPanelIndex === PANEL_COLLECTED && isExpanded ? style.expanded : ''].join(' ')}  />
-							</a>
-							<LocalLabel localizeKey='links.data.title'>Information that may be used.</LocalLabel>
-							<a onClick={this.handleInfo(PANEL_PURPOSE)}>
-								<ChevronIcon class={[style.expand, selectedPanelIndex === PANEL_PURPOSE && isExpanded ? style.expanded : ''].join(' ')}  />
-							</a>
-							<LocalLabel localizeKey='links.purposes.title'>Purposes for storing information.</LocalLabel>
+                                    <LocalLabel localizeKey='links.data.description'>
+                                        Information that may be used:
+                                        <ul>
+                                            <li>Type of browser and its settings</li>
+                                            <li>Information about the device's operating system</li>
+                                            <li>Cookie information</li>
+                                            <li>Information about other identifiers assigned to the device</li>
+                                            <li>The IP address from which the device accesses a client's website or
+                                                mobile application
+                                            </li>
+                                            <li>Information about the user's activity on that device, including web
+                                                pages and mobile apps visited or used
+                                            </li>
+                                            <li>Information about the geographic location of the device when it accesses
+                                                a website or mobile application
+                                            </li>
+                                        </ul>
+                                    </LocalLabel>
+                                </div>
+                                <div
+                                    class={[style.option, selectedPanelIndex === PANEL_PURPOSE && isExpanded ? style.expanded : ''].join(' ')}>
+                                    <a
+                                        onClick={this.handleInfo(PANEL_PURPOSE)}
+                                        style={{color: theme.textLink}}
+                                    >
+                                        <ChevronIcon/>
+                                        Purposes for storing information.
```

Comment:
Translation missing, it should be `<LocalLabel localizeKey='links.purposes.title'>Purposes for storing information.</LocalLabel>`